src/ui/buttons/button.jsx

```diff
@@ -58,7 +58,13 @@ export function Button({ fundingSource, style, multiple, locale, env, fundingEli
     } = style;
 
     if (multiple && i > 0) {
-        color = secondaryColors[color] || secondaryColors[BUTTON_COLOR.DEFAULT] || colors[0];
+        if (secondaryColors[color]) {
```

Comment:
We should also check if `colors.indexOf(secondaryColors[color]) !== -1` right?

Reply:
Yes. Thank you

Reply:
Yes. Thank you.